scripts/deploy-federation.sh

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env bash
+
+# This script automates deployment of a federation - as documented
+# in the README - to the current kubectl context.
+#
+# If using minikube, a cluster must be started prior to invoking this
+# script:
+#
+#   $ minikube start
+#
+# This script depends on crinit, kubectl, and apiserver-boot being
+# installed in the path.  These binaries can be installed via the
+# download-test-binaries.sh script, which downloads to ./bin:
+#
+#   $ ./scripts/download-test-binaries.sh
+#   $ export PATH=$(pwd)/bin:${PATH}
+#
+# To redeploy federation from scratch, prefix the deploy invocation with the deletion script:
+#
+#   # WARNING: The deletion script will remove federation and cluster registry data
+#   $ ./scripts/delete-federation.sh && ./scripts/deploy-test-federation.sh <image>
```

Comment:
./scripts/deploy-test-federation.sh should be ./scripts/deploy-federation.sh

Reply:
Done.